src/js/components/AppPageComponent.jsx

```diff
@@ -343,11 +343,8 @@ var AppPageComponent = React.createClass({
     var volumes = AppsStore.getVolumes(model.id);
 
     var tabs = state.tabs.filter(tab =>
-      tab.text !== "Volumes" || (
-        model.container != null &&
-        model.container.volumes != null &&
-        model.container.volumes.length > 0
-      ));
```

Comment:
I have to align with @Poltergeist , even if there is no task is running, the volumes tab should be shown imho. Reasons for that were already told and it's also a separation of concerns at the UX level.

If the new if-conditon means that this is not the case, then this should be changed to the ol behavior imho.

Could be done in this PR or also in another PR later. What do think @pierlo-upitup ?